pkg/project/namespace.go

```diff
@@ -82,7 +82,7 @@ func (ns *Namespaced) Type() string {
 
 // IsValid returns whether or not the namespace is set sufficiently.
 func (ns *Namespaced) IsValid() bool {
-	return ns != nil && ns.Owner != "" && ns.Project != ""
+	return ns != nil && ns.Project != ""
```

Comment:
Sneak.

Reply:
?

Reply:
It looks like a change from another PR, so I thought it was being sneaked in.

Reply:
Ah. I guess it sort of is in that I found this issue while running the `state use` test suite to see if the bug was fixed (the issue being that `state use shell cli` would not work).